### ISSUE TITLE
XT - Rendering jitter in the UI with 150% zoom level

### ISSUE THREAD

**Issue by xard-dev** [surge-synthesizer/surge#5565]
**Bug Description:**
Rendering jitter in the UI with 150% zoom level.

**Surge Version**
Surge XT 0.99.main.007a427

* Version:
* Plugin Type: LV2
* Bitness: 64-bit

**Reproduction Steps:**
1. Set zoom level 150%.
2. Adjust Oscillator parameters.

**Expected Behavior:**
The text should stay in place while the sliders are moved.

**Screenshots:**
![Peek 2021-12-06 13-09](https://user-images.githubusercontent.com/65303522/144836567-12f43739-4e83-4ce9-97c8-a56ca0dba25c.gif)

**Computer Information (please complete the following information):**
 - OS: Ubuntu 20.04 LTS
 - Host: Carla
 - Version: 2.1.0

**Additional Information:**
A zoom level of 140% or 160% can be used as a mitigation. Also zoom levels 125%, 175% and 200% do not have this issue.

**Comment by baconpaul** [comment]
Wow is it possible to check if this happens in the standalone also? But I’ll find some time this week to boot up my ubuntu box and see if I can repro.

**Comment by xard-dev** [comment]
Confirmed on standalone!

**Comment by mkruselj** [comment]
Just for reference - cannot confirm this on Windows.

**Comment by baconpaul** [comment]
yeah nor on mac
i think what is happening is at 150% the position of the text comes out at *exactly* 3.5 pixels or what not and each repaint is getting a slightly different side of round. My fix I think will be to position the text at rectangle translated by -epsiolon in that case, but i need to boot my linux box and repro it first (along with the nan crash)

**Comment by baconpaul** [comment]
I can reproduce this on my ubuntu box no problem at all.
Which is good. That means I can fix it even if the fix ends up being a bit 'tacky' (I have a hacky fix in mind...)

**Comment by baconpaul** [comment]
So I am seriously considering forcing linux to 149% if you set zoom to 150. I can't for the life of me see why this is happening.

**Comment by xard-dev** [comment]
I wonder what mechanism the JUCE uses for font rendering. Maybe the issue could be there?

Though setting a 149% or 149.9% would be good enough unless something else distorts badly.

**Comment by baconpaul** [comment]
Yeah OK so i have a diff ready to go that does 150->149 on linux and from my inspection it both renders correctly and fixes the problem, so I'm going to merge that but not close this issue; rather i'll drop it out of the xt1 milestone for us to de-hackify in the future.
